babel-watch.js

```diff
@@ -294,6 +299,18 @@ function restartApp() {
   childApp = app;
 }
 
+const delay = program.delay || 1000;
+let timeout;
+function restartApp() {
+  if (timeout) {
+    clearTimeout(timeout);
+  }
+  timeout = setTimeout(function() {
```

Comment:
can you please use an arrow function instead to match the convention with the rest of the file